lib/order.rb

```diff
@@ -0,0 +1,27 @@
+class Order
+
+  attr_reader :basket, :total, :menu
+
+  def initialize
+    @basket = Hash.new(0)
+    @total = 0
+    @menu = Menu.new
+  end
+
+  def add_to_basket(item, quantity)
+    raise "Item not in menu" unless menu.items.include?(item)
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)